iOS_SDK/OneSignalSDK/Source/OSMessagingController.m

```diff
@@ -142,6 +143,11 @@ - (instancetype)init {
 
 - (void)didUpdateMessagesForSession:(NSArray<OSInAppMessage *> *)newMessages {
     self.messages = newMessages;
+    
+    // Cache if messages passed in are not null, this method is called from on_session for
+    //  new messages and cached when foregrounding app
```

Comment:
can the "new message" be formated to the same line as cache 😂  pls, or is some convention?

Reply:
Im confused

Reply:
the indentation of the second line is not alined to the first one

Reply:
it should be, this is an entire sentence not a new thought